.devcontainer/postCreate.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Opam stuff post install
+opam init -y --bare --disable-sandboxing git+https://github.com/rescript-lang/opam-repository
+opam switch create 5.2.0 --packages ocaml-option-static
+opam install -y dune cppo=1.6.9 js_of_ocaml-compiler=5.8.1 ocamlformat=0.26.2 ounit2=2.2.7 reanalyze=2.25.1
```

Comment:
This line can be removed as you are doing

```sh
opam install . --deps-only -y
```

below. This has the advantage that we don't need to specify the dependency versions here.